sql/mob_spawn_points.sql

```diff
@@ -19661,8 +19661,8 @@ INSERT INTO `mob_spawn_points` VALUES (17145896,'Goobbue','Goobbue',4243,-188.04
 INSERT INTO `mob_spawn_points` VALUES (17145897,'Goblin_Flagman','Goblin Flagman',4235,-193.697,24.110,-247.884,24);
 INSERT INTO `mob_spawn_points` VALUES (17145898,'Thunder_Elemental','Thunder Elemental',4307,-211.506,24.267,-235.815,97);
 INSERT INTO `mob_spawn_points` VALUES (17145899,'Gadfly','Gadfly',4227,-207.367,24.488,-176.884,16);
-INSERT INTO `mob_spawn_points` VALUES (17145900,'Gadfly','Gadfly',4274,-210.437,24.406,-151.231,22);
-INSERT INTO `mob_spawn_points` VALUES (17145901,'Peiste','Peiste',4227,-201.892,24.499,-195.025,165);
+INSERT INTO `mob_spawn_points` VALUES (17145900,'Peiste','Peiste',4274,-210.437,24.406,-151.231,22);
```

Comment:
Model/group needs to be changed, not names.

As of Sept 2018, `17145900` is actually a Gadfly.